test/unit/test_describe.py

```diff
@@ -33,29 +33,35 @@ def test_describe(
         if expid not in _EXPIDS:
             continue
         exp = autosubmit_exp(expid)
-
+    
         basic_config = mocker.MagicMock()
+        # TODO: Whenever autosubmitconfigparser gets released with BasicConfig.expid_dir() and similar functions, the following line and a lot of mocks need to be removed. This line is especially delicate because it "overmocks" the basic_config mock, thus making the last assertion of this file "assert f'Location: {exp.exp_path}' in log_result_output" a dummy assertion. The rest of the test is still useful.
```

Comment:
Can you break lines so it's shorter? It adds a warning (in the IDE, or ruff, other linters), but long lines like this one if modified are also harder to see what changed in them -- https://sembr.org/